### ISSUE TITLE
ERROR 1146 (42S02) at line 1: Table 'magento.core_config_data' doesn't exist

### ISSUE THREAD

**Issue by kaynenh** [paliarush/vagrant-magento#109]
I'm seeing an Error 1146 when I attempt to install. I'm not sure where to go with this as I've tried reinstalling based on the information in this thread: #91 

Here are the verbose errors. I'm assuming that "Table 'magento.core_config_data' doesn't exist [Unknown source file]" is the line where it's failing:

```
[2016-12-06 18:40:00]             >  Clearing directories containing temporary files [/vagrant/scripts/guest/m-clear-cache]
[2016-12-06 18:40:00]     >  Enabling Magento cron jobs [/vagrant/scripts/guest/m-reinstall]
[2016-12-06 18:40:00]     >  ERROR 1146 (42S02) at line 1: Table 'magento.core_config_data' doesn't exist [Unknown source file]
[2016-12-06 18:40:00]     >  Configuring search engine according to config.yaml [/vagrant/scripts/guest/configure_search_engine]
[2016-12-06 18:40:00]         >  Search engine configuration was not changed (custom search engine is not available in Magento CE). [/vagrant/scripts/guest/configure_search_engine]
[2016-12-06 18:40:00]     >  Changing Magento configuration to allow execution of functional tests [/vagrant/scripts/guest/change_magento_config_for_functional_tests]
[2016-12-06 18:40:00]         >  Updating Magento config: cms/wysiwyg/enabled = disabled [/vagrant/scripts/guest/update_magento_config]
[2016-12-06 18:40:00]         >  Updating Magento config: admin/security/admin_account_sharing = 1 [/vagrant/scripts/guest/update_magento_config]
[2016-12-06 18:40:00]         >  Updating Magento config: admin/security/use_form_key = 0 [/vagrant/scripts/guest/update_magento_config]
[2016-12-06 18:40:00]     >  Checking if Magento frontend is accessible at 'http://magento2.vagrant46/' [/vagrant/scripts/guest/m-reinstall]
[2016-12-06 18:40:00]     >  ERROR 1146 (42S02) at line 1: Table 'magento.core_config_data' doesn't exist
ERROR 1146 (42S02) at line 1: Table 'magento.core_config_data' doesn't exist
ERROR 1146 (42S02) at line 1: Table 'magento.core_config_data' doesn't exist [Unknown source file]
[2016-12-06 18:40:00]     >  Magento frontend is not accessible at 'http://magento2.vagrant46/' after installation. Please scan output above and logs for errors [/vagrant/scripts/guest/m-reinstall]
```

**Comment by paliarush** [comment]
Please try to execute `bash ./m-reinstall` right after getting this failure and look at the output in `./log/debug.log`. It should give you the details why exactly Magento installation failed.

**Comment by kaynenh** [comment]
Thanks! That's what I needed. I found out that the command line version of php I was running didn't have mcrypt or intl installed.

After checking that log, I was getting a composer error during install at `./bin/magento setup:install` that said I needed to run `composer install` in the application directory. When I did that in bash, I saw the errors that mcrypt and intl weren't installed. After installing them for php 5.6 (I'm using homebrew so it was just as simple as running `brew install homebrew/php/php56-mcrypt` and `brew install homebrew/php/php56-intl`), I had to rerun `./m-reinstall` twice to get this to work (I'm assuming something cached didn't get removed on the first run). Anyway, everything installed and I have a working magento install finally. Thanks again for the help!
